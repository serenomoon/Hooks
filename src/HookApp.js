import React from 'react'

export const HookApp = () => {
  return (
    <div>
        <h1>HookApp</h1>
    </div>
  )
}
