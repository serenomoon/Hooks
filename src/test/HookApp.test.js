import React from 'react';
import {shallow} from 'enzyme';
import '@testing-library/jest-dom';
import { HookApp } from '../HookApp';

describe('Test "HookApp.js"', () => {


    test('should show correctly snapshot', () => {

        const wrapper = shallow( <HookApp /> );

        expect(wrapper).toMatchSnapshot();
        
    });
} )