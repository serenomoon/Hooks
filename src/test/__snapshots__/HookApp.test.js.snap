// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test "HookApp.js" should show correctly snapshot 1`] = `
<div>
  <h1>
    HookApp
  </h1>
</div>
`;
